ddtrace/internal/datadog/profiling/dd_wrapper/src/exporter.cpp

```diff
@@ -370,30 +383,23 @@ Profile::start_sample(unsigned int nframes)
 void
 Profile::push_frame_impl(std::string_view name, std::string_view filename, uint64_t address, int64_t line)
 {
-    if (cur_frame >= lines.size() || cur_frame >= locations.size())
+    if (cur_frame >= locations.size())
         return;
 
     name = insert_or_get(name);
     filename = insert_or_get(filename);
 
-    lines[cur_frame] = {
-      .function =
-          {
-              .name = to_slice(name),
-              .system_name = {},
-              .filename = to_slice(filename),
-              .start_line = 0,
-          },
-      .line = line,
-  };
-
-    locations[cur_frame] = {
+    locations[cur_frame] = ddog_prof_Location{
         {},
+        ddog_prof_Function{
+          to_slice(name),
+          {},
+          to_slice(filename),
+          line,
+        },
         address,
-        { &lines[cur_frame], 1 },
-        false,
+        line,
```

Comment:
Same `line` twice

Reply:
Indeed, thank you.  The `line` quantity in function is actually the start _line_ in the source of the Function object.  At one point I thought, "we don't have any value for this, so let's just send the current line."  In reality, we don't even  use the start_line value in the BE or UI anywhere, so I just zero out the Function line now.